tests/reftests/list-coinstallable.test

```diff
@@ -0,0 +1,7449 @@
+697eab693df71fb665a546e779348a9a823dea2c
```

Comment:
One weird thing here is that I've tried to use other hashes but they weren't able to show the same behaviour :/ @AltGr do you have any idea why?

Reply:
nevermind i found a way. It must've been something that made `opam-0install.0.4.2` unavailable with https://github.com/ocaml/opam-repository/commit/7371c1d9 (this behaviour of `opam list --coinstallable-with` is quite surprising and annoying to work with)